Source/Core/Core/HW/AddressSpace.cpp

```diff
@@ -303,6 +304,7 @@ struct CompositeAddressSpaceAccessors : Accessors
 
 struct SmallBlockAccessors : Accessors
 {
+  SmallBlockAccessors() : alloc_base() {}
```

Comment:
`alloc_base` can just be assigned nullptr directly in the class (along with the size being set to zero). This constructor can then just be defaulted.